README.md

```diff
@@ -24,14 +24,15 @@ For non Sublime development a bunch of other IDE plugins are available [here](ht
 ## Log style
 To try and keep consistency across our systems
 type and namespace
-query    -> payload
-reply    -> payload
-request  -> payload
-response -> payload
-log      -> system
-responseTime -> system
-system   -> error
-thirdPartySystem -> error
+
+		query    -> payload
```

Comment:
It looked like the original author of the docs was trying to keep that text formatted. Also it was hard to read, so by indenting it is now code formatted and easy to read. Happy to take it out, I did almost put it into a table ;)